src/GUI/Valuetable.java

```diff
@@ -0,0 +1,126 @@
+package GUI;
+import Core.ParametricFunction;
+import Core.PolynomialFunction;
+import Core.Vector2D;
+
+import javax.swing.*;
+import javax.swing.table.DefaultTableModel;
+import javax.swing.table.TableModel;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class Valuetable extends JDialog{
+    private Double[][] valuetablefigures = null;
+    private String[][] valuetablestrings = null;
+    private ParametricFunction parametricFunction = null;
+    private JTable table = null;
```

Comment:
More than half of these states are only used locally if i see that right so it would be better to move the declaration to a local spectrum if they are not used globally

Reply:
Done